app/models/assignment.rb

```diff
@@ -19,6 +19,8 @@ class Assignment < ApplicationRecord
   validates :client_id, uniqueness: { scope: :volunteer_id, message: I18n.t('assignment_exists') }
   validates :state, inclusion: { in: STATES.map(&:to_s) }
 
+  scope :default_order, (-> { order(created_at: :desc) })
```

Comment:
rename this to `default_scope` and then you can change this in the controller:
```ruby
@assignments = @q.result.default_order.paginate(page: params[:page])
# to
@assignments = @q.result.paginate(page: params[:page])
```

Reply:
@casaper when I do this, I got following error with the tests:
```VolunteerApplicationsTest#test_new_volunteer_application:
ArgumentError: You tried to define a scope named "default_scope" on the model "Assignment", but Active Record already defined a class method with the same name.```

Reply:
Although this was not introduced in this PR, I think @casaper meant [this](https://apidock.com/rails/ActiveRecord/Base/default_scope/class).

Reply:
yup. @casaper these changes were already introduced in another branch and have been merged already to dev. I have it my branch due to of rebasing. 
so I would prefer to leave this like it is. 
And btw, I found an interesting article on [why it is not recommended to use default scope](https://www.ombulabs.com/blog/ruby/rails/best-practices/why-using-default-scope-is-a-bad-idea.html)